.github/workflows/connector-tests.yml

```
@@ -0,0 +1,130 @@
+            cdk_extra: vector-db-based
+          - connector: destination-motherduck
+            cdk_extra: sql
+    if: >
+      ( github.event_name == 'pull_request' && needs.cdk_changes.outputs.src == 'true' && github.event.pull_request.head.repo.fork != true
+      ) || github.event_name == 'workflow_dispatch'
+    name: "Check: '${{matrix.connector}}' (skip=${{needs.cdk_changes.outputs[matrix.cdk_extra] == 'false'}})"
+    steps:
+      - name: Abort if extra not changed (${{matrix.cdk_extra}})
+        id: no_changes
+        if: ${{ matrix.cdk_extra != 'n/a' && needs.cdk_changes.outputs[matrix.cdk_extra] == 'false' }}
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)